### PR TITLE
Surrender Verb and Emote Overhaul (fixed)

### PR DESCRIPTION
**Pull request by mudzbe**
### About The Pull Request
 New surrender verb where the user is paralyzed to prevent warcrimes(?). An animation is played when entering and exiting this state to alert others. The old surrender emote links to this action. A white flag is animated when surrendering and a exclamation mark is displayed when exiting (the same animation used with boxes and stealth implants). Handcuffing paralyzed (also surrendered) people is faster. The surrender action is logged for admins. (Repeat of #4356 which was closed because I was doing other things)

Here is the white flag codersprite:
image

### Why It's Good For The Game
Before the emote was barely used, this will make the emote more useful to promote RP and perhaps reduce violence. Commonly innocent people are gunned down with disablers or bashed with stunbatons to ensure they comply with being handcuffed. Crewmembers are also extremely racist against syndicate agents and display a 'take no prisoners' attitude, regardless of any peaceful intent. The emote will ensure innocent people are protected from such abuses, and will make plain the cooperative and peaceful attitude of the user. alternatively it makes war crimes more visible. The improved visibility of the emote will also inform people of it's existence, I've barely seen it used.

### Changelog
:cl:
add: New surrender verb where the user is paralyzed to prevent warcrimes(?). An animation is played when entering and exiting this state to alert others
tweak: The old surrender emote is linked to the new surrender verb
tweak: Handcuffing paralyzed (surrendered) people is faster
/:cl:

### CODE REVIEW

File: austation/code/modules/mob/living/carbon/human/human_defines.dm [hunk 1]
```diff
@@ -2,3 +2,4 @@
 	var/ring_type = RING_DISABLED
 	var/ring_engraved = ""
 	var/ringsoff = FALSE
+	var/surrender_time = 0
```
Comment (TheFakeElon) [inline austation/code/modules/mob/living/carbon/human/human_defines.dm:5]: There's a better way to do this with timers and their ID's but this is fine for now if you don't want to mess with that stuff, it's certainly more deserving of memory than 3 rings variables 😆

File: code/game/objects/items/handcuffs.dm [hunk 2]
```diff
@@ -43,6 +43,7 @@
 /obj/item/restraints/handcuffs/attack(mob/living/carbon/C, mob/living/user)
 	if(!istype(C))
 		return
+	var/t = 40 - ((C.has_status_effect(STATUS_EFFECT_PARALYZED)) * 20) //austation -- Makes handcuffing paralzyed people faster
```
Comment (TheFakeElon) [inline code/game/objects/items/handcuffs.dm:46]: multipart suggestion
```suggestion
```

File: code/game/objects/items/handcuffs.dm [hunk 3]
```diff
@@ -57,7 +58,7 @@
 								"<span class='userdanger'>[user] is trying to put [src.name] on you!</span>")
 
 			playsound(loc, cuffsound, 30, 1, -2)
-			if(do_mob(user, C, 40) && (C.get_num_arms(FALSE) >= 2 || C.get_arm_ignore()))
+			if(do_mob(user, C, t) && (C.get_num_arms(FALSE) >= 2 || C.get_arm_ignore())) //austation -- Makes handcuffing paralzyed people faster
```
Comment (TheFakeElon) [inline code/game/objects/items/handcuffs.dm:61]: Could have applied it to the t variable instead but this has less austation vs bee line diffs :)
```suggestion
			if(do_mob(user, C, (C.has_status_effect(STATUS_EFFECT_PARALYZED) ? 20 : 40)) && (C.get_num_arms(FALSE) >= 2 || C.get_arm_ignore())) //austation -- Makes handcuffing paralzyed people faster
```

**Review by TheFakeElon — COMMENTED** [review]
looks good for the most part!

File: austation/code/modules/mob/living/carbon/human/human_defines.dm (hunk 1 above)
Comment (mudzbe) [reply austation/code/modules/mob/living/carbon/human/human_defines.dm:5]: Me computer is broken, she’ll be right

**Comment by mudzbe** [comment]
> looks good for the most part! Thought I'm not sure why you're checking/interacting with paralysis and surrendered, you should only need to use with the latter

Surrendered has no physical effect, paralysis is what immobilises people. It’s use however is so that if a paralysed person uses the surrender verb, it doesn’t count their-self as unsurrendering, potentially removing their paralysis, which doesn’t make sense, amongst triggering an animation

**Comment by TheFakeElon** [comment]
Just make the surrendered effect paralyze instead of applying both effects

**Comment by TheFakeElon** [comment]
Pending changes

**Comment by MCterra10** [comment]
closing due to stale

**Comment by DrDuckedGoose** [comment]
This poor fucking PR

**Comment by mudzbe** [comment]
I see all your mates here laughing

**Pull request merged**